src/external/hauser_parabolic_smoother/CMakeLists.txt

```diff
@@ -11,4 +11,5 @@ set_target_properties("${PROJECT_NAME}_external_hauserparabolicsmoother"
 )
 target_compile_options("${PROJECT_NAME}_external_hauserparabolicsmoother"
   PUBLIC ${AIKIDO_CXX_STANDARD_FLAGS}
+  PRIVATE -w
```

Comment:
If I'm correct, this is to ignore warnings from `${PROJECT_NAME}_external_hauserparabolicsmoother` target. Since it's not an external target, we cannot use `include_directories()` with `SYSTEM` option.